nexus/auth/src/authn/external/session_cookie.rs

```diff
@@ -13,13 +13,15 @@ use chrono::{DateTime, Duration, Utc};
 use dropshot::HttpError;
 use http::HeaderValue;
 use nexus_types::authn::cookies::parse_cookies;
+use omicron_uuid_kinds::{ConsoleSessionKind, TypedUuid};
```

Comment:
nitpicky, not important: why not use the somewhat more concise:
```suggestion
use omicron_uuid_kinds::ConsoleSessionUuid;
```

(which is a type alias for `TypedUuid<ConsoleSessionKind>`)